packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -311,15 +318,15 @@ class _DefaultPub implements Pub {
         mapFunction: filterWrapper, // may set versionSolvingFailed, lastPubMessage
         environment: await _createPubEnvironment(context, flutterRootOverride),
       );
-      String message;
-      switch (code) {
-        case 69: // UNAVAILABLE in https://github.com/dart-lang/pub/blob/master/lib/src/exit_codes.dart
+      String? message;
+      if (retry && attempts < _kPubMaxRetryAttempts) {
+        if (code == _kPubExitCodeUnavailable) {
           message = 'server unavailable';
-          break;
-        default:
-          break loop;
+        }
+      }
+      if (message == null) {
+        break;
```

Comment:
since we no longer use the `loop` label, we can remove it?

Reply:
done